webextensions/api/action.json

```diff
@@ -280,6 +280,29 @@
             }
           }
         },
+        "getUserSettings": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/action/getUserSettings",
+            "support": {
+              "chrome": {
+                "version_added": "91"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "116"
+              },
+              "firefox_android": {
+                "version_added": "116",
```

Comment:
Should we add `"partial_implementation": true`?

Reply:
Good point, it does seem to meet the criteria e.g. the sub-feature deviates from the specification in a way that may cause significant compatibility problems.

Reply:
Hmmm...Instead of partial_implementation, maybe it would be more clear if we removed the note and instead added specific compat info for the userSettings object, and then marked `isOnToolbar` as unsupported on Android.

Sorta like this: https://github.com/mdn/browser-compat-data/blob/92db750cdcc48fd095de10b6723394aa2c45bc25/webextensions/api/search.json#L48-L132

Reply:
I did contemplate taking that approach in the first place. However, I thought the note whas a more explicit way of stating that the object is empty. @Rob--W thoughts?

Reply:
For now there is only one supported property, but in the future there may be more.

I am okay with any of the discussed approaches. The property-level detail of the return value may be more future-proof though.